pkg/asset/installconfig/password.go

```diff
@@ -17,20 +20,48 @@ func (a *password) Dependencies() []asset.Asset {
 	return []asset.Asset{}
 }
 
-// Generate queries for the password from the user.
+// Generate the kubeadmin password
 func (a *password) Generate(asset.Parents) error {
-	p, err := asset.GenerateUserProvidedAsset(
-		a.Name(),
-		&survey.Question{
-			Prompt: &survey.Password{
-				Message: "Password",
-				Help:    "The password of the cluster administrator. This will be used to log in to the console.",
-			},
-		},
-		"OPENSHIFT_INSTALL_PASSWORD",
-	)
+	p, h, err := generateRandomPasswordHash()
+	if err != nil {
+		return err
+	}
 	a.Password = p
-	return err
+	a.PasswordHash = h
+	return nil
+}
+
+// generateRandomPasswordHash generates a hash of a random ASCII 14 char string with at least
+// one digit and one special character.
+func generateRandomPasswordHash() (string, []byte, error) {
+	rand.New(rand.NewSource(time.Now().UnixNano()))
+	const (
+		lowercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+		uppercase = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
Let us avoid ambiguous characters such as `0` and `O` - even with the reduced entropy we still have ~100 bits.

Reply:
took out `0` `O` `1` `l`